src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRow1099Processor.java

```diff
@@ -1127,6 +1131,14 @@ private void processExclusionsAndAmounts(ResultSet rs, TransactionDetailRow deta
                         .append(paymentDateP.value).append(';')
                         .append(docNumberP.value).append(';')
                         .append(docLineNumberP.value).toString());
+        
+        //Determine if there is override of tax box due to the payment reason
+        if (getPaymentReason1099BoxService().isPaymentReasonMappedTo1099Box(paymentReasonCodeP.value)) {
+			String mappedBox = getPaymentReason1099BoxService().getPaymentReason1099Box(paymentReasonCodeP.value);
+			LOG.info("Overriding to tax box " + mappedBox + "  because of payment reason " + paymentReasonCodeP.value + " for vendor " + vendorNameForOutput);
+			overrideTaxBox = summary.getBoxNumberConstant(mappedBox);

```

Comment:
You should verify whether the tax managers want payment-reason-based tax box overrides to ignore other exclusions that may have been discovered for the given transaction. Your current setup appears to be doing so because of its re-use of the overrideTaxBox variable. If the tax managers do not want this behavior, then I would suggest using a different variable for tracking such overrides. Also, I believe payment reason overrides should *not* overwrite any of those derived from the summary.transactionOverrides field just a couple lines earlier.